static/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -54,14 +54,13 @@ function EventOrGroupExtraDetails({
 
   const showReplayCount =
     organization.features.includes('session-replay') && projectCanLinkToReplay(project);
-  const hasEscalatingIssuesUi = organization.features.includes('escalating-issues');
 
   return (
     <GroupExtra>
-      {!hasEscalatingIssuesUi && inbox && (
+      {inbox && (
```

Comment:
oops missed this earlier, but the `!hasEscalatingIssuesUi` block can be deleted.